src/src.sln

```diff
@@ -0,0 +1,643 @@
+﻿
```

Comment:
Do we need this file?
I'm actually thinking of creating separate projects inside folders, as was recently done for Microsoft.Extensions - https://github.com/dotnet/runtime/tree/main/src/libraries/Microsoft.Extensions.Logging.Abstractions

Reply:
It was added in previous commit